Make Zapier Tutorial more friendly (#4521)
umbraco/UmbracoDocs: Tutorials/Connecting-Umbraco-Forms-and-Zapier/index.md

![Zapier Google Sheets Possible Fields](images/zapierGoogleSheetsPossibleFields.png)

> The fields showing are all columns in your spreadsheet that have a name in row 1. To map the input data to the different fields in the spreadsheet, you click on the “Type or insert…” field and choose which data to put in that field.

Reviewer: If we put the steps into a list like this, we avoid the too long sentence. What do you think?

```suggestion
The fields showing are all columns in your spreadsheet that have a name in row 1. To map the input data to the different fields in the spreadsheet follow these steps:

* Select the “Type or insert…” field.
* Choose which data to put in the field.
```
